bin/cli.js

```diff
@@ -0,0 +1,273 @@
+#!/usr/bin/env node
+'use strict';
+
+const program = require('commander');
+const common = require('@metarhia/common');
+const karma = require('karma');
+const yaml = require('yaml').default;
+const path = require('path');
+const fs = require('fs');
+
+const splitOpts = v => v.split(',');
+
+const cliOptions = [
+  ['--exclude <patterns>', 'Exclude tests patterns', splitOpts, []],
+  ['--browsers <values>', 'Browsers to run', splitOpts, []],
+  ['--reporter <value>', 'Reporter name'],
+  ['--log-level <value>', 'Log level'],
+  ['--run-todo', 'Run todo tests'],
+  ['--unresolved-module <value>', 'Unresolved modules handler'],
```

Comment:
It is not a handler.
```suggestion
  ['--unresolved-module <value>', 'Unresolved modules handling strategy (\'none\', \'fail\')'],
```